src/assets/utils/birthDayValidation.js

```diff
@@ -0,0 +1,13 @@
+const isAdult = (inputDate) => {
+  const today = new Date();
+  const eighteenYearsAgo = new Date(
+    today.getUTCFullYear() - 18,
+    today.getMonth(),
+    today.getUTCDate()
+  );
+  const birthday = new Date(inputDate);
+
+  return birthday > eighteenYearsAgo ? false : true;
```

Comment:
`birthday > eighteenYearsAgo` already returns boolean value so you can skip ternary operator :)
```suggestion
  return birthday > eighteenYearsAgo;
```

Reply:
corrected